pennylane_lightning/core/src/lightning.qubit.toml

```diff
@@ -0,0 +1,79 @@
+schema = 1
+
+[device]
+name = "lightning.qubit"
+version = "v0.31.0"
```

Comment:
Does manually changing this for each minor release seem reasonable to you?

Reply:
Also, this is not the current version of lightning. `__version__ = "0.34.0-dev10"`

Reply:
What I am arguing here is that it doesn't make sense to pin the pennylane-lightning's version in the toml file. For one, the version is available in the `__version__` dunder method. And as a second note, what we care about is the gates that are implemented by the device, independent of the version.

Reply:
The "version" will always change predictably. I don't think this is comparable to "operations", for example.
You can always query the device for its version (or check the [__version__](https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/core/_version.py) file). When you have several versions of a device, all devices will bring this information.
I would prefer to have the version been queried from the device, or if we have already in place a way to automatically update the toml files. It might be possible to do it by expanding the [dev_version_script](https://github.com/PennyLaneAI/pennylane-lightning/blob/master/.github/workflows/dev_version_script.py).

Reply:
Let's actually revisited this when the perf squad wants to use these toml files for supporting the Python bindings (`lightning_qubit.py`, `lightning_kokkos.py`, and `lightning_gpu.py`) using the new PL Device API. cc: @mlxd @AmintorDusko @vincentmr

Reply:
Removing it and closing this comment, if people feel the need, please reopen this conversation.